wordpress-mobile/WordPress-iOS
# Finalizes splitting out app settings from account settings

Fixes #5118 

Adds missing icon & analytics event to App Settings. Moves About into App Settings.

To test:
1. Launch app and set up a single site, self-hosted scenario. Ensure App Settings only exists.
2. Launch app and set up a single (or multiple) site, .com scenario. Ensure both Account and App Settings exists.

![simulator screen shot apr 11 2016 2 26 07 pm](https://cloud.githubusercontent.com/assets/373903/14439684/0bc82e92-fff2-11e5-90db-1045d44a97d4.png)

![simulator screen shot apr 11 2016 2 26 01 pm](https://cloud.githubusercontent.com/assets/373903/14439686/0e71c9dc-fff2-11e5-9424-144eb971aab6.png)


Needs review: @SergioEstevao, @rickybanister
